service/src/main/kotlin/fi/espoo/evaka/attendance/RealtimeStaffAttendanceController.kt

```diff
@@ -288,7 +294,7 @@ class RealtimeStaffAttendanceController(private val accessControl: AccessControl
         clock: EvakaClock,
         @RequestBody body: StaffAttendanceBody
     ) {
-        if (body.anyAttendanceAfter(HelsinkiDateTime.atStartOfDay(clock.today().plusDays(1)))) {
+        if (body.anyAttendanceAfter(clock.now())) {
             throw BadRequest("Attendances cannot be in the future")
```

Comment:
I would love to see an integration tests that triggers this error. And all other errors where validation logic was changed.
e.g.
```
assertThrows<BadRequest> {
            realtimeStaffAttendanceController.upsertStaffAttendances(...
```

Reply:
👍  added some tests